internal/reader/urlcleaner/urlcleaner.go

```diff
@@ -95,6 +112,20 @@ var trackingParamsOutbound = map[string]bool{
 	"ref": true,
 }
 
+var TrackParamsPrefixes = []string{
```

Comment:
This variable doesn't need to be exported outside the package. Look at your previous PRs :) https://github.com/miniflux/v2/pulls?q=is%3Apr+unexport

```suggestion
var trackingParamsPrefixes = []string{
```